### DISCUSSION TITLE
Support for Oracle to store the results

### DISCUSSION THREAD

**Discussion by rthangam** [GoogleChrome/lighthouse-ci#554] [Q&A]
storage.sqlDialect
One of mysql, postgres, or sqlite. sqlite in a local file on disk has been sufficient for most use cases.

Is it possible to use Oracle database to store the results ?

**Comment by patrickhulce** [comment]
It is not supported. LHCI uses [`sequelize`](https://github.com/sequelize/sequelize/) under the hood which does not support Oracle as a dialect.

Conceptually it is certainly possible to build out a new Oracle-backed storage method without *a ton* of effort. If you're interested in exploring this yourself you would just need to satisfy the [StorageMethod API](https://github.com/GoogleChrome/lighthouse-ci/blob/main/packages/server/src/api/storage/storage-method.js).

**Reply by rthangam** [reply]
Thanks for the information. I will try it out.

**Reply by rthangam** [reply]
Looks like oracledb npm module can be used as well. Is it possible to store the json results and generate html files when required ?
